### PR TITLE
Untangle ColorPicker includes

### PR DESCRIPTION
**Pull request by KoBeWi**
Forward-declares stuff from `color_picker.h` + some minor cleanup.
The other file changes were indirect includes that had to be resolved.

### CODE REVIEW

**Comment by Mickeon** [comment]
So we're starting to favour forward-declarations more now?

**Comment by KoBeWi** [comment]
We started long ago, no? Not all old code was "fixed" though and I was just doing something with ColorPicker. All new files are expected to use forward-declarations though.

**Review by Calinou — APPROVED** [review]

File: scene/gui/color_picker.h [hunk 1]
```diff
@@ -110,11 +106,11 @@ class ColorPicker : public VBoxContainer {
...
-	static Ref<Shader> wheel_shader;
-	static Ref<Shader> circle_shader;
-	static Ref<Shader> circle_ok_color_shader;
-	static List<Color> preset_cache;
-	static List<Color> recent_preset_cache;
+	inline static Ref<Shader> wheel_shader;
+	inline static Ref<Shader> circle_shader;
+	inline static Ref<Shader> circle_ok_color_shader;
+	inline static List<Color> preset_cache;
+	inline static List<Color> recent_preset_cache;
```
Comment (Repiteo) [inline scene/gui/color_picker.h:113]:
```suggestion
	static inline Ref<Shader> wheel_shader;
	static inline Ref<Shader> circle_shader;
	static inline Ref<Shader> circle_ok_color_shader;
	static inline List<Color> preset_cache;
	static inline List<Color> recent_preset_cache;
```

**Comment by Repiteo** [comment]
Thanks!

**Pull request merged**